readthedocs-requirements.txt

```
@@ -13,4 +13,5 @@ setuptools>=41.2
 multimethod>=1.2
 findiff
 jupyter-sphinx
-pytest
\ No newline at end of file
+pytest
+sphinxcontrib.bibtex
```

The package is sphinxcontrib-bibtex:
```suggestion
sphinxcontrib-bibtex
```